### ISSUE TITLE
improve descriptions in mapping file

### ISSUE THREAD

**Issue by kMutagene** [nfdi4plants/isa-ro-crate-profile#33]

**Comment by floWetzels** [comment]
@kMutagene Can you specify what kind of improvements you wish to see? @HLWeil and I are unsure whether we should include descriptions for every property (this would be redundant with the profile description) or only specific properties where it is not easy to see.

**Comment by kMutagene** [comment]
Been a while since i discussed this with @HLWeil but i think in the mapping file we wanted to see more reasoning/explanation behind the non-trivial mappings, i think a prime candidate here is `processSequence -> about`. I'd agree that there does not need to be more info about something like `description -> description`.

**Comment by floWetzels** [comment]
I agree that for such mappings a precise description of reasoning can be helpful. But where do we draw the line? For example, is `technologyType -> measurementMethod` a trivial mapping or not? How about `studies -> hasPart` and `dataFiles -> hasPart`?
Also, all properties are described in the profile itself, e.g. processSequence/about is described by "The experimental processes performed in this study". I agree that this does not include reasoning on why this mapping was chosen, but again, there is some fuzziness here.

**Comment by kMutagene** [comment]
I think that this can be closed as part of #48, i'd still love to have a preface text in the mapping explaining some choices but i would say the current state is good enough for now.
